routes/web.php

```diff
@@ -13,6 +13,29 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
+Route::get('/', function () {return view('welcome');
 });
+
+Route::get('login', function() {return view('/auth/login');
+});
```

Comment:
Interesting formatting. I wolud reccomend to format these Routes like so:

```suggestion
Route::get('login', function() {
    return view('/auth/login');
});
```